80_machinelearning/ML1100_World_Population.md

```
@@ -0,0 +1,59 @@
+# Projekt World Population
+
+### Projektmål
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)